<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Team</class>
 <widget class="QWidget" name="Team">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1612</width>
    <height>674</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>151</width>
     <height>106</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="name">
      <property name="styleSheet">
       <string notr="true">font-size:30px;
</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="score">
      <property name="styleSheet">
       <string notr="true">color:red;font-size:20px;</string>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="status_2">
        <property name="text">
         <string>status:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="status">
        <property name="text">
         <string>stat</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>1551</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,1,0">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QTableWidget" name="duel">
        <column>
         <property name="text">
          <string>id</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>status</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="duel_btn">
        <property name="text">
         <string>Duel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>Question Number</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>price</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>max-score</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>score</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>bought</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="buy">
          <property name="text">
           <string>buy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="solve">
          <property name="text">
           <string>solve</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="random">
          <property name="text">
           <string>buy random</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <item>
       <widget class="QTableWidget" name="games">
        <column>
         <property name="text">
          <string>name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>cost</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>score</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="play">
        <property name="text">
         <string>play</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>209</width>
     <height>139</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="A_exchange"/>
        </item>
        <item>
         <widget class="QPushButton" name="A_exchange_btn">
          <property name="text">
           <string>exchange A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLineEdit" name="b_exchange"/>
        </item>
        <item>
         <widget class="QPushButton" name="B_exchange_btn">
          <property name="text">
           <string>exchange B</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>A currency</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="A_amount"/>
        </item>
        <item>
         <widget class="QLabel" name="A_2">
          <property name="text">
           <string>$</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="change_a">
          <property name="text">
           <string>change</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>B currency</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="B_amount"/>
        </item>
        <item>
         <widget class="QLabel" name="A_3">
          <property name="text">
           <string>$</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="change_b">
          <property name="text">
           <string>change</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="duel_frame">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#duel_frame{border: 2px solid black;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>you have an active battle</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>attacker:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="attacker">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>defender</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="defender">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>30</y>
      <width>82</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <widget class="QPushButton" name="loose">
       <property name="text">
        <string>loose</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="win">
       <property name="text">
        <string>win</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QFrame" name="invitation_frame">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>100</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#invitation_frame{border: 2px solid black;}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>you have a duel invitaions</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>82</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <widget class="QPushButton" name="accpet">
       <property name="text">
        <string>accept</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reject">
       <property name="text">
        <string>reject</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="attacker_invitation">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>137</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>65</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>attacker:</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="game_frame">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>82</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <widget class="QPushButton" name="loose_game">
       <property name="text">
        <string>loose</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="win_game">
       <property name="text">
        <string>win</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>you have an active game</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>114</width>
      <height>18</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>game:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="game_name">
       <property name="text">
        <string>game name</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
